src/spatialNavigation.js

```diff
@@ -661,9 +667,7 @@ class SpatialNavigation {
       /**
        * Otherwise, trying to focus something by coordinates
        */
-      const sortedXChildren = sortBy(children, (child) => child.layout.left);
-      const sortedYChildren = sortBy(sortedXChildren, (child) => child.layout.top);
-      const {focusKey: childKey} = first(sortedYChildren);
+      const {focusKey: childKey} = getNearestChild(children);
```

Comment:
They seems a bit different, in the previous implementation it was selecting the child most top-left while now it's selecting the child closest to 0,0 
But it's not a big issue since the most top-left point is 0,0
By the way I'd change the function name to something like `getChildClosestToOrigin`

Reply:
Hi, yeah, you right, but the previous implementation worked so strangely.
For example:
```js
const children = [
    { layout: { left: 1000000, top: 0 } },
    { layout: { left:  1, top: 0.1 } }
]; 

// First item will be { layout: { left:  1, top: 0.1 } } It's a correct result
const sortedXChildren = sortBy(children, (child) => child.layout.left); 

const sortedYChildren = sortBy(sortedXChildren, (child) => child.layout.top);
//  Now, the first one is { layout: { left: 1000000, top: 0 } }.
// But we can't see this item. Right? :)

 const result = first(sortedYChildren); // { layout: { left: 1000000, top: 0 } }
```

Reply:
you right :+1:

Reply:
🙂
About the new function name. No problem, I'll change it)